app/models/concerns/turbo/broadcastable.rb

```diff
@@ -296,8 +310,11 @@ def broadcast_rendering_with_defaults(options)
       options.tap do |o|
         # Add the current instance into the locals with the element name (which is the un-namespaced name)
         # as the key. This parallels how the ActionView::ObjectRenderer would create a local variable.
-        o[:locals] = (o[:locals] || {}).reverse_merge!(model_name.element.to_sym => self)
-        o[:partial] ||= to_partial_path
+        # if the html or inline options are passed it will skip setting a partial from #to_partial_path
+        unless o.include?(:html) || o.include?(:inline)
+          o[:locals] = (o[:locals] || {}).reverse_merge!(model_name.element.to_sym => self)
```

Comment:
Thinking through this further, I believe we'd only want to conditionally omit the `partial:` option when `html:` or `inline:` is present. continuing to accept and forward along `locals:` feels appropriate.

```ruby
o[:locals] = (o[:locals] || {}).reverse_merge!(model_name.element.to_sym => self)

unless o.include?(:html) || o.include?(:inline)
  o[:partial] ||= to_partial_path
end
```

Reply:
This is very valid as you can use the locals inside of `inline:` option